build/config/RepoUtilData.json

```diff
@@ -7,6 +7,7 @@
     "System.Collections.Immutable": "1.1.36",
     "Microsoft.VisualStudio.CoreUtility": [ "14.3.25407", "15.0.25726-Preview5" ],
     "Microsoft.VisualStudio.Editor": [ "14.3.25407", "15.0.25726-Preview5" ],
+    "Microsoft.VisualStudio.GraphModel": [ "14.3.25407", "15.0.25726-Preview5" ],
```

Comment:
@RaulPerez1 if you rebase off of this PR you shouldn't need any changes here https://github.com/dotnet/roslyn/pull/16305